aictl/cli/main.py

```diff
@@ -161,6 +161,24 @@ def t2v(args):
     print(video_path)
 
 
+# Text to Audio
+def t2a(args):
+    from audiocraft.models import MusicGen
+    from audiocraft.data.audio import audio_write
+
+    # TODO add this to args
+    model = MusicGen.get_pretrained("melody")
+
+    # Set the duration
+    model.set_generation_params(duration=int(args.duration))
+
+    # Set the description, for now one at a time
+    wav = model.generate([args.prompt])
+
+    # TODO set strategy to arguments
+    audio_write(args.output_path, wav[0].cpu(), model.sample_rate, strategy="loudness")
```

Comment:
It seems like we need to install `ffmpeg` in order to use this function. Would it be needed with the `scipy` method of saving the audio file? Example:
```python
import scipy

sampling_rate = model.config.audio_encoder.sampling_rate
scipy.io.wavfile.write("musicgen_out.wav", rate=sampling_rate, data=audio_values[0, 0].numpy())
```

Reply:
Correct me if I am wrong but ffmpeg is required for audiocraft. Is there a reason why we don't want ffmpeg?


From audiocraft git :  https://github.com/facebookresearch/audiocraft
**Note: Please make sure to have [ffmpeg](https://ffmpeg.org/download.html) installed when using newer version of torchaudio.'**

Reply:
Resolved this conversation because it looks like we don't need `ffmpeg` to run the `audio_write` function.